data-prepper-plugins/blocking-buffer/src/test/java/com/amazon/dataprepper/plugins/buffer/blockingbuffer/BlockingBufferTests.java

```diff
@@ -77,6 +81,15 @@ public void testInsertNull() throws TimeoutException {
         blockingBuffer.write(null, TEST_WRITE_TIMEOUT);
     }
 
+    @Test(expected = PayloadSizeOverflowException.class)
```

Comment:
We should update this to JUnit 5 and use assertThrows rather than go deeper into JUnit 4.